contrib/rhcs.md

```diff
@@ -0,0 +1,83 @@
+## Log into Red Hat's registry
+
+The RH Ceph Storage image is based on ubi8. To pull this image on your
+computer, you will need a Red Hat Customer Portal account. Log in with your
+username and password:
+
+```
+$ podman login registry.redhat.io
+Username: **********
+Password: **********
+Login Succeeded!
+```
+
+You will then be able to run `podman pull registry.redhat.io/ubi8/ubi:latest`
+to fetch the base image.
+
+## Composing the Dockerfile
+
+The `ceph-container` project uses a series of template files to create the
+final `Dockerfile` that developers can build or commit to dist-git. This
+command generates that downstream Red Hat UBI-based `Dockerfile`:
```

Comment:
nit: mention that it should be done on a stable-x.0 branch

Reply:
Thanks for the suggestion. I've updated it to recommend `stable-6.0` since that corresponds to RH Ceph Storage 5.